CMakeLists.txt

```diff
@@ -52,6 +70,18 @@ if(Intl_FOUND)
 	include_directories(SYSTEM ${Intl_INCLUDE_DIRS})
 endif()
 
+find_program(SH_EXE sh)
+if(NOT SH_EXE)
+	message(FATAL_ERROR "sh interpreter was not found in your path, please install one. On Windows you can get it from here https://gitforwindows.org/")
```

Comment:
This message is pretty specific to Windows; I wonder whether we need to provide a non-Windows message, too?

Reply:
Well, except windows other OS's(Linux, Mac, *BSD) provide sh

Reply:
Or so we assume...

But I guess your message is first general, and then qualifies what to do on Windows. That's definitely good enough for me.